src/splitInteger.test.js

```diff
@@ -1,17 +1,24 @@
+/* eslint-disable max-len */
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it. Find the way to solve it without `eslint-disable`.